### DISCUSSION TITLE
Error 404 when accessing the api

### DISCUSSION THREAD

**Discussion by zrsserr** [nilsteampassnet/TeamPass#4041] [Q&A]
Hello. I have a problem with the api
When reading a folder using Postman, I get a 404 error
The request looks like this:
http://localhost/TeamPass-3.1.0/api/index.php/item/get?id=2&Authorization ="token"
Folder with ID 2 is valid
API usage is enabled
But I changed the source code in the file /var/www/html/TeamPass-3.1.0/api/Model/AuthModel.php
And I commented out the api_key validation, because when I received the token, I got an error, although I copied the generated key from the site. But that's how I got the token
                // check API key
                #if ($inputData['apikey'] !== base64_decode(decryptUserObjectKey($userInfo['user_api_key'], $privateKeyClear))) {
                #    return ["error" => "Login failed4.", "apikey" => "Not valid"];
                #}

**Comment by justin77x** [comment]
bro i also got same error how did you solved
